docs/verifier-syntax.md

```diff
@@ -0,0 +1,73 @@
+## Unit test comment syntax
```

Comment:
I think this should not be duplicated. If we move it here, we should remove the class comment

Reply:
I replaced the doc comment with a link to the new file.